libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -421,4 +421,13 @@ class Strings {
         assertPrints(emptyString.lastOrNull(), "null")
         assertFails { emptyString.last() }
     }
+  
+    @Sample
+    fun replace() {
+        val inputString0 = "Mississippi"
+        val inputString1 = "How can the net amount of entropy of the universe be massively decreased?"
```

Comment:
Let's have a shorter sentence. It will be difficult for documentation readers to find out where exactly replace occurred in a long sentence.

Reply:
I've shortened `inputString1`. If that's not short enough, I can shorten it further.